src/ensembl/production/metadata/grpc/utils.py

```diff
@@ -93,9 +93,13 @@ def create_genome_with_attributes_and_count(db_conn, genome, release_version):
         dataset_attributes=True
     )
     # fetch related assemblies count
-    related_assemblies_count = db_conn.fetch_organisms_group_counts(
+    count_result = db_conn.fetch_related_assemblies_count(
         species_taxonomy_id=genome.Organism.species_taxonomy_id
-    )[0].count
+    )
+
+    related_assemblies_count = 0
+    if len(count_result) > 0:
+        related_assemblies_count = count_result[0].count
```

Comment:
Could be simplified by 
`related_assemblies_count = count_result[0].count if len(count_result) > 0 else 0`
Or from previous comment, the method directly return a count, as its name suggests.

Reply:
I implemented your suggestion above

Reply:
I just did :-)